plugins/aggregators/starlark/README.md

```diff
@@ -0,0 +1,87 @@
+# Starlark Aggregator
+
+The `starlark` aggregator calls a Starlark function to add the metrics to the aggregator,
+then calls another Starlark function to push the metrics returned to the accumulator. Both Starlark functions
+have an argument called `cache` which is a [dict-like][dict] object allowing to keep temporary the metrics to
+aggregate. The `cache` is automatically cleared by the the aggregator.
+
+The Starlark language is a dialect of Python, and will be familiar to those who
+have experience with the Python language. However, there are major [differences](#python-differences).
+Existing Python code is unlikely to work unmodified.  The execution environment
+is sandboxed, and it is not possible to do I/O operations such as reading from
+files or sockets.
+
+The **[Starlark specification][]** has details about the syntax and available
+functions.
+
+## Configuration
+
+```toml
+[[aggregators.starlark]]
+  ## The Starlark source can be set as a string in this configuration file, or
+  ## by referencing a file containing the script.  Only one source or script
+  ## should be set at once.
+  ##
+  ## Source of the Starlark script.
+  source = '''
+def add(cache, metric):
+  cache["last"] = metric
+
+def apply(cache):
+  return cache.get("last")
+'''
+
+  ## File containing a Starlark script.
+  # script = "/usr/local/bin/myscript.star"
+
+  ## The constants of the Starlark script.
+  # [aggregators.starlark.constants]
+  #   max_size = 10
+  #   threshold = 0.75
+  #   default_name = "Julia"
+  #   debug_mode = true
+```
+
+## Usage
+
+The Starlark code should contain a function called `add` that takes a cache as first argument and a metric as
+second argument. The function will be called with each metric to add them to the cache, and doesn't return anything.
+
+```python
+def add(cache, metric):
+  cache["last"] = metric
```

Comment:
As I explained in a previous comment, if I do so we will need to define the state in all the Starlark scripts ([like in this example](https://github.com/influxdata/telegraf/blob/master/plugins/processors/starlark/testdata/compare_metrics.star#L10-L12)) which I wanted to avoid